bspump/elasticsearch/data_feeder.py

```diff
@@ -0,0 +1,47 @@
+import orjson
+
+"""
+Data feeders to be used in ElasticSearchSink.
+"""
+
+
+async def data_feeder_create_or_index(items):
+	for _id, data in items:
+		yield b'{"create":{}}\n' if _id is None else orjson.dumps(
+			{"index": {"_id": _id}}, option=orjson.OPT_APPEND_NEWLINE
+		)
+		yield data
+
+
+async def data_feeder_create(items):
+	for _id, data in items:
+		yield b'{"create":{}}\n' if _id is None else orjson.dumps(
+			{"create": {"_id": _id}}, option=orjson.OPT_APPEND_NEWLINE
+		)
+		yield data
+
+
+async def data_feeder_index(items):
+	for _id, data in items:
+		yield b'{"index":{}}\n' if _id is None else orjson.dumps(
+			{"index": {"_id": _id}}, option=orjson.OPT_APPEND_NEWLINE
```

Comment:
The same issue as above.

Reply:
It is a condition, if ID is None or if it is specified.